modules/serverpod_auth/serverpod_auth_server/lib/src/business/users.dart

```diff
@@ -101,6 +101,23 @@ class Users {
     return userInfo;
   }
 
+  /// Updates a users name, returns null if unsuccessful.
+  static Future<UserInfo?> changeFullName(
+      Session session, int userId, String newFullName) async {
```

Comment:
Suggestion(NIT): Add a `,` after the last parameter to improve formatting.
```suggestion
  static Future<UserInfo?> changeFullName(
      Session session, int userId, String newFullName,) async {
```

Reply:
Makes sense, Will fix.

Reply:
@SandPod , the other methods in this class doesn't have a trailing comma. Should I update them as well now when I'm doing some chores?

Reply:
Yes, please! 🙏